draft-ietf-ice-pac.xml

```diff
@@ -66,31 +66,38 @@
         for Network Address Translator (NAT) traversal for UDP-based communication.
       </t>
       <t>
-        Typically, when using ICE, endpoints will exchange candidate addresses,
-        form a list of candidate pairs, and then test each candidate pair to see
-        if connectivity can be established. If the test for a given pair fails,
-        it is marked accordingly, and if all pairs have failed, the overall
-        ICE process is considered to have failed.
+        When using ICE, endpoints will exchange candidiates, form one or more
+        lists (checklists) of candidate pairs, and then test the pairs to
+        determine whether connectivity can be established. There is a checklist
+        for each data stream associated with the session. If connectivity for
+        a data stream cannot be established, the state of the checklist associated
+        with the data stream is set to Failed. Once the state of all checklists
+        have been set to Failed, the ICE agent sets the state of the ICE session
+        to Failed. Based on local policy, the ICE agent can also set the state of 
+        the ICE session to Failed even if the state of only some of the checklists
+        have been set to Failed.
       </t>
```

Comment:
It seems like the previous text was sufficient, and shorter and easier to read.  Can we just keep it?

Reply:
There was at least one technical issue in the original text, that I wanted to fix:

The text says that an agent declares ICE failure once the state of all checklists are set to Failed. While that is true, I wanted to point out that an agent can declare failure earlier than that, e.g., if a checklist that represents a data stream required for the session fails.

But, it's not major, so it can be fixed later.

Reply:
I wanted to cover the fact that an agent may declare ICE failure before all checklists have failed. But, it can be done later.